Assets/Plugins/GameJolt/readme.txt

```diff
@@ -0,0 +1,18 @@
+Asset Setup
+1. Select "Edit > Project Settings > Game Jolt API" (this will create your GameJolt API settings object)
+2. Edit the settings in your inspector window, you MUST enter your "Game ID" and "Private Key".
```

Comment:
I think it would be worth mentioning that those can be obtained on the GameJolt website.